frontend/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```

Comment:
I think this might override the current readme, is it possible to change this to our current one?